go/validation/tests/bigquery.py

```diff
@@ -25,7 +25,7 @@ class BigQueryQuirks(model.DriverQuirks):
     # BigQuery doesn't really have a public facing version, so use the client
     # version instead
     vendor_version = "cloud.google.com/go/bigquery v1.72.0"
-    short_version = "gcloud"
+    short_version = "1.72.0"
```

Comment:
This gets weird because BigQuery isn't really versioned per se and this is reporting the client version

Reply:
We can iterate on it